examples/farm/platform/platform.yaml

```
@@ -0,0 +1,15 @@
+# Copyright (c) CloudCover
+# SPDX-License-Identifier: Apache-2.0
+
+platforms:
+  - name: postgres
+    repo_url: https://github.com/your/repo
+    repo_directory: examples/farm/platform/
+    ref_label: v1.0 # The label can be tag/branch/commit
+    label_type: 2 # 1=branch, 2=tag, 3=commit
```

```suggestion
    label_type: tag # one of: 'branch', 'tag', 'commit'
```

this yaml format should be user friendly, so we should let users put the type in string. internally we should change it to enum for efficiency if needed.